synapse/lib/ingest.py

```diff
@@ -119,17 +119,23 @@ def _fmt_lines(fd,gest):
 def _fmt_json(fd,info):
     yield json.loads( fd.read() )
 
+def _fmt_jsonl(fd,info):
+    for line in fd.readlines():
```

Comment:
the readlines() method will read the entire content of the file into a list all at once ( which would preclude ingest of files too large to lift into RAM).  Plz replace with:
```
for line in fd:
```
( like the _fmt_lines function above )